app/main.py

```diff
@@ -1,2 +1,54 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.threshold = int(self.capacity * (2 / 3)) + 1
+        self.length = 0
+        self.hash_table = [[]] * self.capacity
+
+    def __setitem__(self, key: object, value: object) -> None:
+        if self.length == self.threshold:
+            self.resize()
+        index = hash(key) % self.capacity
+
+        while True:
+            if not self.hash_table[index]:
+                self.hash_table[index] = [key, value]
+                self.length += 1
+                break
+            if key == self.hash_table[index][0]:
+                self.hash_table[index][1] = value
+                break
+            index = (index + 1) % self.capacity
+
+    def __getitem__(self, key: object) -> object:
+        index = hash(key) % self.capacity
+
+        while True:
+            if self.hash_table[index]:
+                if key == self.hash_table[index][0]:
+                    return self.hash_table[index][1]
+            else:
```

Comment:
and redundant else as well

Reply:
@OleksandrPozden 

Oleksandr, hello!
Could you, please, check the updated solution )